### ISSUE TITLE
Crash on Startup after Hotfix on v.1.63

### ISSUE THREAD

**Issue by TheSilverBat220** [WopsS/RED4ext#26]
How would I be able to fix this issue? Would I need to manually fix it myself or wait for an update?:

[2023-06-24 19:07:50.801] [RED4ext] [info] RED4ext (v1.14.0) is initializing...
[2023-06-24 19:07:50.801] [RED4ext] [info] Game patch: Unknown patch (3.0.72.54038)
[2023-06-24 19:07:50.801] [RED4ext] [info] Product version: 1.63
[2023-06-24 19:07:50.801] [RED4ext] [info] File version: 3.0.72.54038
[2023-06-24 19:07:50.801] [RED4ext] [error] This game patch (Unknown patch (3.0.72.54038)) is not supported
[2023-06-24 19:07:50.801] [RED4ext] [error] The current version of RED4ext supports only the following patch(es): 1.63 (3.0.72.45053)

**Comment by gir489returns** [comment]
There's already been a commit to this repo like 2 days ago to fix this, and a github action compiled it.

**Comment by DedSecTV** [comment]
I'm a noob to this, how do I install the fix?

**Comment by cogula** [comment]
Please, he's not the only one with this problem, I also don't know how to solve it using just the github build. Could you help us ???

**Comment by unrealxqt** [comment]
I think this is the build artifact: https://github.com/WopsS/RED4ext/actions/runs/5358504352

**Comment by Teddy-Kun** [comment]
I agree that there should probably a new release. Most regular users have no idea how to use build artifacts.

**Comment by Jeoshua** [comment]
[2023-06-25 21:16:54.048] [RED4ext] [warning] TweakXL (version: 1.1.5) is incompatible with the current patch (1.63 Hotfix 1). This version of the plugin was compiled for patch 1.62 Hotfix 1

That fragment doesn't cover the hotfix.

**Comment by Jeoshua** [comment]
[2023-06-25 21:16:54.048] [RED4ext] [warning] TweakXL (version: 1.1.5) is incompatible with the current patch (1.63 Hotfix 1). This version of the plugin was compiled for patch 1.62 Hotfix 1

Keep in mind to update all dependencies.

**Comment by dragonzkiller** [comment]
The update will be released when it is ready. Please be patient in the time being
